Add investing query parameters and +37 currencies support (#25)
DavideViolante/investing-com-api: README.md

async function main () {
  try {

>     const response = await investing('currencies/eur-usd', 3600, 24, '1-day') //EURxUSD quotes of last 24h (input, interval, candle_count, period)

Reviewer: ```suggestion
    const response1 = await investing('currencies/eur-usd')
    const response2 = await investing('currencies/eur-usd', 3600, 24, '1-day') // With optional params
```

### Response
```js
[
  { date: 1623812400000, value: 1.1093 },
  { date: 1623816000000, value: 1.1054 },
  { date: 1623819600000, value: 1.1025 },
  { date: 1623823200000, value: 1.1018 },
  ...
]
```

### Available inputs

> - [mapping.js](https://github.com/DavideViolante/investing-com-api/blob/master/mapping.js)
> - interval (in milliseconds quotes interval)
> - candle_count (maximum number of results)
> - period (n-day or n-month or n-year)

Reviewer: ```suggestion
- input (String), see [mapping.js](https://github.com/DavideViolante/investing-com-api/blob/master/mapping.js)
- interval (Number in seconds), data interval
- candleCount (Number) Max number of results
- period (String) time window: n-day, n-month or n-year where n is a number
```

### Author
- [Davide Violante](https://github.com/DavideViolante/)

> ### Contributor

Reviewer: ```suggestion
### Contributors
```
